main.tf

```diff
@@ -34,6 +34,9 @@ data "aws_ami" "example" {
   most_recent = true
 }
 
+# The default tags configured for the default provider
+data "aws_default_tags" "default" {}
```

Comment:
Would this be better served in another existing file or a new file?

Reply:
It won't normally even be around unless you're creating EC2 instances.  If you are then it would probably make sense to add this bit to your `locals.tf`.  For this example, though, I wanted to highlight how to handle this confusing case of `volume_tags` so I think it makes sense to keep the code together.

Eventually this whole issue will probably go away, since folks are asking that provider default tags also be applied to `volume_tags` and it seems reasonable (and more sane) to do so.